pkg/sql/pg_catalog.go

```diff
@@ -3315,6 +3353,110 @@ func addPGTypeRow(
 	)
 }
 
+func addPGClassRow(
```

Comment:
nit: this should be named a bit more specifically. maybe `addPGClassRowForCompositeType`, and this function could use a comment. it would be good to mention that it does not add rows for enum types